extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js

```diff
@@ -10,6 +10,7 @@ define('ext.wikia.adEngine.slot.service.megaAdUnitBuilder', [
 	var dfpId = '5441',
 		megaSlots = [
 			'INVISIBLE_SKIN',
+			'INCONTENT_PLAYER',
```

Comment:
It means that we're changing INCONTENT_PLAYER to MEGA outside wgCountries too. Is it fine?

Reply:
Ok, I'll do this only if railPlayer is enabled.